### PR TITLE
:mag: Add custom tracking for online tasks

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles-pr-130 in nhsuk-dev succeeded (http://profiles-pr-130.dev.beta.nhschoices.net)

File: public/js/custom-hj.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ jQuery(function($) {
     $('.gp-book-online').on('click', function () {
       hj('tagRecording', ['Book Online']);
     });
+    $('.repeat-prescription-online').on('click', function () {
```
Comment (ajthomascouk) [inline public/js/custom-hj.js:9]: On `gp-finder` we used `.on('touchstart click')` as there were some instances where click wasn't working on mobile.

**Review by ajthomascouk — CHANGES_REQUESTED** [review]
change above

File: public/js/custom-hj.js (hunk 1 above)
Comment (st3v3nhunt) [reply public/js/custom-hj.js:9]: A touch client must support `click` events as the basic interaction and under the covers a `click` event is also being triggered. From my testing I've not found any problems. Is there is a specific test case I can run?

Comment (ajthomascouk) [reply public/js/custom-hj.js:9]: to it was on iphone 5 for a reveal, not button. so I guess its not an issue here. I'll approve it.

**Review by ajthomascouk — APPROVED** [review]

**Pull request merged**